src/components/application_manager/test/resumption_sql_queries_test.cc

```diff
@@ -113,9 +113,11 @@ class ResumptionSqlQueriesTest : public ::testing::Test {
   static const int timeStamp2;
 
   static void SetUpTestCase() {
-    db_ = new SQLDatabase();
+    db_ = new SQLDatabase(utils::dbms::Persistent::IN_MEMORY);
```

Comment:
@BSolonenko any reason why changed to in-memory? Maybe it makes sense to wrap this in __QNX__ ifdef?

Reply:
@AKalinich-Luxoft The default constructor creates an in-memory database. It is not obvious. This was taken for better readability of the code.